src/groupeddataframe/splitapplycombine.jl

```diff
@@ -194,24 +194,27 @@ function _combine_process_agg(@nospecialize(cs_i::Pair{Int, <:Pair{<:Function, S
                               gd::GroupedDataFrame,
                               seen_cols::Dict{Symbol, Tuple{Bool, Int}},
                               trans_res::Vector{TransformationResult},
-                              idx_agg::Union{Nothing, AbstractVector{Int}})
+                              idx_agg::Ref{Union{Nothing, Vector{Int}}})
```

Comment:
`idx_agg` should never be `nothing` here AFAICT (I know it is my code, but `idx_agg` is the most tricky part here so I think it is better to narrow down the signature to catch bugs if they are present).